scripts/check-merge-conflicts.sh

```diff
@@ -0,0 +1,11 @@
+result=$(git ls-tree --full-tree --name-only -r HEAD | while read file_path; do
```

Comment:
Do we need a license here?

Reply:
In theory a license is required. However TiCDC doesn't check this in `scripts/check-copyright.sh` and most bash files also have no license provided.
Maybe we should add check and add missing license

Reply:
> In theory a license is required. However TiCDC doesn't check this in `scripts/check-copyright.sh` and most bash files also have no license provided.
> Maybe we should add check and add missing license

Let me file an issue to land it.